src/doing/pr/create_pr.py

```diff
@@ -129,6 +133,7 @@ def cmd_create_pr(
     command += f'--draft "{str(draft).lower()}" '
     command += f'--work-items "{work_item_id}" '
     command += f'--source-branch "{branch_name}" '
+    command += f'--target-branch "{default_branch}" '
```

Comment:
This sets the target branch, but I think we don't define where to branch from. I believe we need to change line 120 as well.

Reply:
The branch_name is the name of the new (to be created) branch with the prefix issueid in it.

Reply:
that is determined on lines 120/121